CFC_DataCollector/trip_generator/tour_model.py

```diff
@@ -1,89 +1,104 @@
+# >>> tues = Day(1, home_start_tues)
+
+# >>> days = [mon, tues]
+# >>> week = TourModel("naomi", days)
+# >>> tm_for_week = week.build_tour_model()
+
+# I know this seems like alot, but you shouldnt really be typing any of this out by hand
+# Maybe there is a better way to do this...
+
 class Location(object):
 
-    """ A class representing a location in a user's tour model. """
-
-    def __init__(self, name):
-        self.name = name
-        self.successors_counter = Counter( )
-
-    def add_to_successors(self, loc, weight=None):
-        """ 
-        This function adds successor locations to the current location along with the probability that the user will travel to that place next. 
-
-        You can input a dictionary with locations as keys and weights as values
-        For instance, if you are most likely to go home after work, and half as likely to go to safeway, 
-        and a fourth as likely to go pick up your kids, you would set up the succesors like this:
-        
-        >>> work = Location('work')
-        >>> home = Location('home')
-        >>> safeway = Location('safeway')
-        >>> pick_up_kids = Location('kids')
-        >>> work_successors = {home : 4, safeway : 2, pick_up_kids : 1}
-        >>> work.add_to_successors(work_successors)
-
-        If all are equally likely you can simply input them all in a list:
-        >>> work_successors = [home, safeway, pick_up_kids]
-        >>> work_successors.add_to_successors(work_successors) 
-
-        And if you want to add a place after originally adding locations you can do so by putting in the location and weight:
-        >>> burritos = Location('burritos')
-        >>> work.add_to_successors(burritos, 100)
-
-        The counter class is a dictionary type class, of which great documentation is provided in util.py. 
-        We use it here as an easy way to get the next state based on the historic probability of user going to that place next.
-
-        """
-        if isinstance(loc, dict):
-            for location, w in loc.iteritems():
-                self.successors_counter[location] = w
-        elif type(loc) == Location:
-            if weight is not None:
-                self.successors_counter[loc] = weight
-            else:
-                self.successors_counter[loc] = 1
-        elif type(loc) == list:
-            for location in loc:
-                self.successors_counter[location] = 1
-        else:
-            raise TypeError("You can not input the location as type %s, please use a dictionary, list or single location and weight" % type(loc))
-
-    def increment_weight_of_successor(self, successor):
-        self.successors_counter[successor] += 1
-
-    def add_n_to_weight_of_successor(self, successor, n):
-        self.successors_counter[sucessor] += n
-
-    def set_weight_of_successor(self, sucessor, weight):
-        self.successors_counter[sucessor] = weight
+    def __init__(self, name, hour, day):
```

Comment:
I am not sure whether you want to have the location have the time or the edge have the time. If you think about this as a graph, you have classes for the nodes, but not the edges. But in our discussion, we had assumed that we would have decorations on the edges (in other words, the time that you go from work to lunch is different from the time you go from work to home)